libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -299,6 +299,26 @@ const AP_Param::GroupInfo AP_GPS::var_info[] = {
     AP_GROUPINFO("DRV_OPTIONS", 22, AP_GPS, _driver_options, 0),
 #endif
 
+    // @Param: COM_PORT
+    // @DisplayName: GPS physical COM port
+    // @Description: The physical COM port on the connected device
```

Comment:
Can we clarify that this parameter is only used by the Septentrio?

Reply:
Done